woocommerce/woocommerce-android
# Custom amounts UI flow

Closes: #9938 


### Description

This PR introduces the UI flow for Custom Amounts, allowing navigation from the order creation screen to the custom amounts screen. It's important to note that the business logic is not yet implemented; this PR solely focuses on the necessary UI components and navigation for custom amounts.

### Testing instructions

1. Navigate to the `Orders` tab
2. Click on the "+" button to navigate to the order creation screen.
3. Click on the "Enter Custom Amounts" button in the `products` section
4. Ensure you see a `Custom Amounts` dialog
5. Enter the amount and ensure the numbers only keyboard appears
6. Enter a custom name and ensure the text keyboard appears
7. Press back and ensure the Custom Amounts dialog disappears

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/1331230/476dd74a-c89b-46ca-aa3e-cd6301de7a21
